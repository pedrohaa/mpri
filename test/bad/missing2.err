File "bad/missing2.f", line 11, characters 2-29:
A case for the data constructor Eq is missing.
Hypotheses:
var_1_ = a
var = b
