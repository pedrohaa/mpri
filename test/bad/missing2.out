forall a b c. eq a b -> c
